source/about/subscription.rst

```diff
@@ -39,46 +39,6 @@ System administrators can view user status in the System Console and activate an
 
 Talk to a `Mattermost Expert <https://mattermost.com/contact-sales/>`_ to learn more.
 
-Mattermost Nonprofit License program
```

Comment:
@Combs7th - Are you open to updating this docs PR to replace this deleted content with your updated content from https://github.com/mattermost/docs/pull/7740?